### PR TITLE
🔧 touch activity table border none

### LINKED ISSUE #6306: Activity sections styling
image

- font weight to regular
- remove lines on every row, separator should be only between events

### PR DESCRIPTION
**Pull request by roiLeo**
- close #6306 

Screenshot 2023-07-19 at 17-35-06 wallpaper #2 KodaDot - NFT Market Explorer

### CODE REVIEW

**Review by preschian — COMMENTED** [review]
netlify build down? maybe try to trigger again?

File: pnpm-lock.yaml
Comment (preschian) [inline pnpm-lock.yaml:1]: some changes in this file? expected?

Comment (roiLeo) [reply pnpm-lock.yaml:1]: Idk why, i've made this PR after Nuxt3 migration branch so I had to remove node_modules & pnpm.lock

**Comment by roiLeo** [comment]
Ok google how to uncommit file

**Pull request closed without merging**